### PR TITLE
feat(env.sh): Take a configuration from file

### LINKED ISSUE #36: Load existing vars from config file CLI option
Option on `env.sh` script, load existing vars from ENV.

We should load [this existing vars](https://github.com/GluuFederation/gluu-passport-testing/blob/7f60f51ee277d459f18ceaa73f31a7dc399b2887/env.sh#L39) from ENV.

Something like `--load-from-env`, other suggestions are welcome.

### PR DESCRIPTION
**Pull request by kdhttps**
Seperate configuration file to manage configurations

closes #36

### CODE REVIEW

File: env.sh [hunk 1]
```diff
@@ -14,11 +14,20 @@ run_tests=true;
...
+# -c <file> : to load config from file 
 
-while getopts ":st" option; do
+while getopts ":stc:" option; do
    case $option in
       s) create_drplet=false;;
       t) run_tests=false;;
+      c)
+        config_file=${OPTARG} 
+        if [ -z ${config_file+x} ];
```
Comment (christian-hawk) [inline env.sh:25]: 1. Can you please check?
If I run `./env.s -c whatever`, output is:
```bash
PLEASE DO NOT INTERRUPT
============================================================================
Configuration is loading from whatever file
./env.sh: line 51: whatever: No such file or directory
```

2. Also, maybe load automatically from file (like `test.conf` if custom configuration file name not passed?)

File: config [hunk 2]
```diff
@@ -0,0 +1,10 @@
+PASSPORT_HOST=t1.techno24x7.com
```
Comment (christian-hawk) [inline config:1]: maybe a more specific name like `tests.conf`?

**Review by christian-hawk — CHANGES_REQUESTED** [review]

File: env.sh (hunk 1 above)
Comment (kdhttps) [reply env.sh:25]: thank you for testing Chris. I'll fix it and take the `test.conf` file as a default

File: config (hunk 2 above)
Comment (kdhttps) [reply config:1]: Agree

**Review by christian-hawk — APPROVED** [review]
Good job @kdhttps  👍🏻

**Pull request merged**